src/main/java/core/Main.java

```diff
@@ -0,0 +1,13 @@
+package core;
+
+import core.strategy.ReportFactory;
+
+public class Main {
+
+    public static void main(String[] args) {
+        ReportFactory reportFactory = new ReportFactory();
+        String filePath = "src/main/java/core/db/dataBase.csv";
+        String pathTo = "test333";
```

Comment:
bad value for files 
is it database or input?? 
is it some unknown file "test333" or report?? 
<img width="614" alt="image" src="https://github.com/mate-academy/jv-fruit-shop/assets/32339836/27739fc8-b561-4316-9a1b-5fdbf5f77b65">